packages/mobile/src/geth/geth.ts

```diff
@@ -182,25 +213,28 @@ export function isProviderConnectionError(error: any) {
     .includes(PROVIDER_CONNECTION_ERROR)
 }
 
-async function ensureStaticNodesInitialized(sync: boolean = true): Promise<boolean> {
-  const { nodeDir } = networkConfig
-  Logger.debug('Geth@ensureStaticNodesInitialized', 'initializing static nodes')
-  let enodes: string | null = null
+async function getStaticNodes(sync: boolean): Promise<string[]> {
+  if (!sync) {
+    return []
+  }
   try {
-    enodes = sync ? await StaticNodeUtils.getStaticNodesAsync(DEFAULT_TESTNET) : '[]'
+    const enodesStr = await StaticNodeUtils.getStaticNodesAsync(DEFAULT_TESTNET)
+    return JSON.parse(enodesStr)
   } catch (error) {
     Logger.error(
       `Failed to get static nodes for network ${DEFAULT_TESTNET},` +
         `the node will not be able to sync with the network till restart`,
       error
     )
-    return false
-  }
-  if (enodes != null) {
-    await writeStaticNodes(nodeDir, enodes)
-    return true
+    throw FailedToFetchStaticNodesError
   }
-  return false
+}
+
+// Writes static nodes to the correct location
+async function initializeStaticNodesFile(staticNodes: string[]): Promise<void> {
+  const { nodeDir } = networkConfig
+  Logger.debug('Geth@ensureStaticNodesInitialized', 'initializing static nodes')
```

Comment:
Nit: tag should be updated
```suggestion
  Logger.debug('Geth@initializeStaticNodesFile', 'initializing static nodes')
```